src/ui/component/button/button.scss

```
@@ -18,6 +18,10 @@
     box-shadow: none;
   }
 
+  &:focus {
+    background-color: $pressed-color;
+  }
```

Shadow can also be removed on focus!
```suggestion
  &:focus {
    box-shadow: none;
    background-color: $pressed-color;
  }
```

---

done, thanks!